src/quadrature/Quadrature__parser.hh

```diff
@@ -33,6 +33,8 @@ public:
 
   Class_Parse_Table();
 
+  virtual ~Class_Parse_Table() { child_.reset(); }
+
```

Comment:
This is the key element of this change. This ensures that child_, a static pointer to the  parsed quadrature, is reset when parsing is complete. Thus the storage will not linger after the client has destroyed his last pointer to the parsed quadrature.